streams/src/test/java/org/apache/kafka/streams/StreamsConfigTest.java

```diff
@@ -650,7 +650,7 @@ public void shouldSpecifyCorrectValueSerdeClassOnError() {
     @Test
     public void shouldThrowExceptionIfMaxInFlightRequestsGreaterThanFiveIfEosEnabled() {
         props.put(StreamsConfig.PROCESSING_GUARANTEE_CONFIG, EXACTLY_ONCE);
-        props.put(ProducerConfig.MAX_IN_FLIGHT_REQUESTS_PER_CONNECTION, 7);
+        props.put(ProducerConfig.MAX_IN_FLIGHT_REQUESTS_PER_CONNECTION, 3);
```

Comment:
Hey @mjsax . Sorry, but I didn't follow this change. How does this test pass when it expects the value to be 7, but it's now 3?

Reply:
Ups -- this change should not be in the PR (I was just trying out locally if 3 is accepted -- the test fails of course for this case).